packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -276,6 +317,61 @@ const Updates = decorate([
             </CardBlock>
           </Card>
         </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('releaseChannels')}</CardHeader>
+            <CardBlock>
+              <div>
+                {_('selectedChannel')}{' '}
+                <h5 className='d-inline-block'>
+                  <span className='tag tag-pill tag-info'>
+                    {xoaConfiguration.channel}
+                  </span>
+                </h5>
+              </div>
```

Comment:
I don't think that's necessary, the selected channel should be visible in the `select`, right?